MAINTAINERS.md

```diff
@@ -14,3 +14,5 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 | Bandini                  | [bandinib-amzn](https://github.com/bandinib-amzn) | Amazon      |
 | Ashwin P Chandran        | [ashwin-pc](https://github.com/ashwin-pc)         | Amazon      |
 | Josh Romero              | [joshuarrrr](https://github.com/joshuarrrr)       | Amazon      |
+| Andrey Myssak            | [andreymyssak](https://github.com/andreymyssak)   |             |
+| Sergey Myssak            | [SergeyMyssak](https://github.com/SergeyMyssak)   |             |
```

Comment:
@andreymyssak @SergeyMyssak Do you want anything for affiliation?

Reply:
Can we add affiliation as "external contributor" similar to https://github.com/opensearch-project/terraform-provider-opensearch/blob/main/MAINTAINERS.md?